client.py

```diff
@@ -169,30 +169,43 @@ def filter_commit_data(commit):
     return repo_dict
 
 
-if __name__ == "__main__":
-    repositories_data = open('./repositories.json').read()
-    repositories = json.loads(repositories_data)
-
-    # Github
+def create_files_for_git():
     """
-    Goes through every repo under github and creates a separate MD file for each one
+    Main GIT function. Takes every Git repo from repositories.json and writes all the commit data of each repo in a
+    separate json file and generates a MD file for each repo as well.
+    :return: the end result is a .json and a .md file for every git repository. can be found inside git_files/
     """
     for repo in repositories["Github"]:
         repository_name = repo
         repository_team = repositories["Github"][repo]["team"]
+        pretty_name = repositories["Github"][repo]["name"]
         git_link = create_git_link(repository_team, repository_name)
         commit_data = get_git_commits(git_link)
         useful_data = filter_commit_data(commit_data)
-        write_commits(useful_data, "changelog.json")
-        create_md_table(repository_name)
-    # Mercurial
+        git_json_filename = "./git_files/" + "{}.json".format(repository_name)
+        write_commits(useful_data, git_json_filename)
+        create_git_md_table(repository_name, pretty_name)
+
+
+def create_files_for_hg():
     """
-    Goes through every repo under mercurial and creates a separate MD file for each one
+    Main HG function. takes every Mercurial repo from repositories.json and writes all the commit data of each repo in a
+    separate json file
+    :return: the end result is a .json file for every HG repository. can be found inside hg_files/
     """
     for repo in repositories["Mercurial"]:
         repository_url = repositories["Mercurial"][repo]["url"]
         repository_push_type = repositories["Mercurial"][repo]["configuration"]["push_type"]
         repository_name = repositories["Mercurial"][repo]["name"]
         hg_changes = get_hg_changes(repository_url, repository_push_type)
-        hg_json_name = "./repositories/" + "{}.json".format(repository_name)
+        hg_json_name = "./hg_files/" + "{}.json".format(repository_name)
         write_commits(hg_changes, hg_json_name)
+
+
+if __name__ == "__main__":
+    repositories_data = open('./repositories.json').read()
+    repositories = json.loads(repositories_data)
+    create_files_for_git()
```

Comment:
This doesn't look right, while the repositories are loaded with data from reading the repositories.json file, the parse of the data to the `create_filess_for_git()` function does not happen explicitly.
This could break and create issues if for some reason a call to the function would happen before the initialisation of `repositories`.
Please fix this, the same story for `create_files_for_hg()'.